prepare_setup.sh

```diff
@@ -270,6 +271,18 @@ packages_install(){
     return $local_status
 }
 
+pip_packages_install(){
+    local_status=0
+    packages="ansible"
+
+    echo ""
+    echo "Installing pip packages $packages ...."
+
+    pip3 install -U pip setuptools
+    pip3 install $packages
```

Comment:
Is it expected always to use the latest ansible version, or should we use a fixed version here?

Reply:
I will pin it to the current version 3.2.0